app/assets/stylesheets/display_tasks.scss

```diff
@@ -4,18 +4,41 @@
 
 body {
   font-family: 'Lato', sans-serif;
+  width: 900px;
+  margin: auto;
 }
 
-table {
-  margin: auto;
+.header {
+  padding-top: 25px;
+  display: inline-block;
 }
 
-th {
-  padding-right: 30px;
+.header a {
+  padding-right: 10px;
+}
+
+table {
+  border-collapse: collapse;
+
+  margin-top: 25px;
+  margin-left: auto;
+  margin-right: auto;
 }
 
 td {
-  padding-right: 30px;
+  border-top: 1px solid #E6E6E6;
+}
+
+tr .completed_task {
```

Comment:
CSS naming conventions use the `-` between words in your CSS classes rather than the `_` like we use in our ruby code